src/app/shared/components/header/header.component.ts

```diff
@@ -9,16 +10,22 @@ import { Router } from '@angular/router'
 })
 export class HeaderComponent {
   public title = 'Sofie'
+  private readonly logger: Logger
 
-  constructor(private readonly router: Router) {}
+  constructor(
+    private readonly router: Router,
+    logger: Logger
+  ) {
+    this.logger = logger.tag('HeaderComponent')
+  }
 
   public navigateHome(): void {
     const routeSegments: string[] = [Paths.HOME]
```

Comment:
Minor thing not really related to this PR:
`routeSegments` makes it sound like that it's actually Segments that are used for routes (which it is not), whereas `segmentRoutes` makes it sound like that it is a Route used for Segments (which it is)

Reply:
Segment is used here as in URL path segments. Should I change this to `pathSegments` or `urlSegments`?

Reply:
Neither ;)

`pathSegments` and `urlSegments` still indicates that it is a subtype of `Segment`, which they are _not_.
`segmentPaths` and `segmentUrls` indicates that they are paths or URLs, which they are.

I have no preference between `segmentRoutes`, `segmentPaths` and `segmentUrls` as long as `segment` isn't the last word.

Reply:
We might be misunderstanding each other. The `segment` part of the name does not refer to our entity `Segment`? Since we have an entity/concept of a `Segment` in this codebase I thought that was what you where referring to...
If you are not referring to the entity I think you should remove `segment` from the name alltogether to avoid further confusion.

If it does refer to our entity, then agree to disagree ;) 

Sidenote: If there is only one path then offcourse you would call it `SegmentPath` instead of `SegmentPaths`. `routeSegments` (which is the original variable name) also indicates that there are multiple routes/paths ;)

Reply:
It is not the entity. I have changed it to `segmentedPath`, though I still think that `pathSegments` better describes the content.